source/docs/software/advanced-controls/controllers/feedforward.rst

```diff
@@ -66,7 +68,7 @@ To calculate the feedforward, simply call the ``calculate()`` method with the de
 ArmFeedforward
 --------------
 
-.. note:: In C++, the ``ArmFeedforward`` class assumes distances are angular, not linear.  The passed-in gains *must* have units consistent with the angular unit, or a compile-time error will be thrown.  ``kS`` and ``kCos`` should have units of ``volts``, ``kV`` should have units of ``volts * seconds / radians``, and ``kA`` should have units of ``volts * seconds^2 / radians``.  For more information on C++ units, see :ref:`docs/software/basic-programming/cpp-units:The C++ Units Library`.
+.. note:: In C++, the ``ArmFeedforward`` class assumes distances are angular, not linear.  The passed-in gains *must* have units consistent with the angular unit, or a compile-time error will be thrown.  ``kS`` and ``kG`` should have units of ``volts``, ``kV`` should have units of ``volts * seconds / radians``, and ``kA`` should have units of ``volts * seconds^2 / radians``.  For more information on C++ units, see :ref:`docs/software/basic-programming/cpp-units:The C++ Units Library`.
```

Comment:
The ArmFeedForward parameter is called kCos, thus there should be a statement of equivalence

Reply:
It is not called that anymore in the tool.

Reply:
If the tool uses one term and the code uses another term, that should be explained.

Reply:
The code doesn't call it `kCos` either, anymore.

Reply:
https://github.com/wpilibsuite/allwpilib/blob/a8f0f6bb904abf13c4212d04e9621dc801442241/wpimath/src/main/java/edu/wpi/first/math/controller/ArmFeedforward.java#L27

Reply:
https://github.com/wpilibsuite/allwpilib/pull/3774